packages/richtext-lexical/src/lexical/ui/ContentEditable.scss

```diff
@@ -25,11 +25,7 @@ $lexical-contenteditable-bottom-padding: 8px;
     }
   }
 
-  .rich-text-lexical--show-gutter
-    > .rich-text-lexical__wrap
-    > .editor-container
-    > .editor-scroller
-    > .editor {
+  .rich-text-lexical > .rich-text-lexical__wrap > .editor-container > .editor-scroller > .editor {
     > .ContentEditable__root {
       padding-left: 3rem;
```

Comment:
This change causes richtext on small screens to now always show a gutter:

Before:
![Screenshot 2025-06-24 at 09 51 10@2x](https://github.com/user-attachments/assets/0a20a4aa-9786-4568-ab6e-71d66d799f98)

After:
![Screenshot 2025-06-24 at 09 50 37@2x](https://github.com/user-attachments/assets/ee4a40ea-0206-4e6b-bc72-6daabab6d295)